src/bg/background.js

```diff
@@ -24,15 +24,24 @@
 
   var getMapInputs = function (url) {
     var matches = /lyrs=([^&]+)(?:.*)&x=([^&]*)(?:.*)&y=([^&]*)(?:.*)&z=([^&]*)/.exec(url);
-    if (!matches || matches.length < 4) {
-      return void 0;
+    if (matches && matches.length >= 5) {
```

Comment:
I think the check for matches.length is redundant, as the result of the regexp should always be null or an array with length 5. I chose to preserve it just in case. I changed it from 4 to 5, though, since the length has to be at least 5 for matches[4] to make any sense below.

Actually, none of the pages I've visited use the old URLs anymore, so it might make sense to remove the entire check for the old URL entirely.